host/traceh.c

```diff
@@ -28,8 +29,9 @@
     sprintf(buffer, format, argument)
 #endif
 
-static char* _log_level_strings[OE_LOG_LEVEL_MAX] =
+const char* oe_log_level_strings[OE_LOG_LEVEL_MAX] =
```

Comment:
```suggestion
const char* const oe_log_level_strings[OE_LOG_LEVEL_MAX] =
```

Explanation: https://www.geeksforgeeks.org/difference-const-char-p-char-const-p-const-char-const-p/ explains about const char*: "You cannot change the value pointed by ptr, but you can change the pointer itself."   Since the array itself should not change, use the 3rd approach there using this suggestion.